package.json

```diff
@@ -108,7 +108,7 @@
   ],
   "dependencies": {
     "decorate-component-with-props": "^1.0.2",
-    "draft-js": "~0.10.0",
+    "draft-js": "~0.10.1",
```

Comment:
What about this dependency change? Because of the change in arity for the plugin functions Draft 0.10.1 seems to be a breaking change. How should I account for that in dependency version?